.github/workflows/sycl-linux-build.yml

```diff
@@ -14,7 +14,7 @@ on:
       build_image:
         type: string
         required: false
-        default: "ghcr.io/intel/llvm/ubuntu2204_build:latest"
+        default: "ghcr.io/intel/llvm/ubuntu2204_build:latest-0300ac924620a51f76c4929794637b82790f12ab"
```

Comment:
Why this? At the very least, it should have a comment in the code explaining it.

Reply:
This is the same as the other places that use this tag.
```shell-session
 $ ag 0300ac92462 -A 1 -B 1
workflows/sycl-linux-build.yml
16-        required: false
17:        default: "ghcr.io/intel/llvm/ubuntu2204_build:latest-0300ac924620a51f76c4929794637b82790f12ab"
18-      build_ref:

workflows/sycl-precommit.yml
81-            runner: '["Linux", "amdgpu"]'
82:            image: ghcr.io/intel/llvm/ubuntu2204_build:latest-0300ac924620a51f76c4929794637b82790f12ab
83-            image_options: -u 1001 --device=/dev/dri --device=/dev/kfd

workflows/sycl-linux-precommit-aws.yml
66-      runner: '["aws_cuda-${{ github.event.workflow_run.id }}-${{ github.event.workflow_run.run_attempt }}"]'
67:      image: ghcr.io/intel/llvm/ubuntu2204_build:latest-0300ac924620a51f76c4929794637b82790f12ab
68-      image_options: -u 1001 --gpus all --cap-add SYS_ADMIN --env NVIDIA_DISABLE_REQUIRE=1
```

 What should the comment say? "Use stable tag 0300ac924620a51f76c4929794637b82790f12ab"? Seems redundant to me